src/Visualization/Python/Render1D.py

```diff
@@ -0,0 +1,277 @@
+        print("Retry without .h5 extension on 'file-prefix'")
+        sys.exit(1)
+
+    return vars(parser.parse_args())
+
+
+def get_h5_files(file_prefix, filename_list):
+    '''
+    Get a list of the h5 files containing the data
+    :return list of h5 files containing volume data
+    '''
+    h5_files = []
+    if file_prefix is not None:
+        h5_file_names = glob.glob(file_prefix + "*.h5")
+        if not h5_file_names:
+            sys.exit("Could not find files with prefix %s in path" %
+                     file_prefix)
+        h5_files = [h5py.File(file_name, 'r') for file_name in h5_file_names]
+
+    else:
+        for file_name in filename_list:
+            try:
+                h5_files.append(h5py.File(file_name, 'r'))
+            except OSError:
+                exit("Could not find file %s in directory %s" %
+                     (file_name, os.path.dirname(file_name)))
+    return h5_files
+
+
+def print_var_names(args):
+    '''
+    Print all available variables to screen
+    :param args:
+    :return: None
+    '''
+
+    file_prefix = args["file_prefix"]
+    filename_list = args["filename_list"]
+    h5files = get_h5_files(file_prefix, filename_list)
+    volfile = h5files[0]["element_data.vol"]
+    obs_id_0 = next(iter(volfile))
+    variables = list(volfile[obs_id_0].keys())
+    variables.remove("connectivity")
+    variables.remove("InertialCoordinates_x")
+    variables.remove("total_extents")
+    variables.remove("grid_names")
+    print("Variables in H5 file:\n[%s]" % ', '.join(map(str, variables)))
+    for h5_file in h5files:
+        h5_file.close()
+    return None
+
+
+def get_data(args):
+    '''
+    Get the data to be plotted
+    :return: the array of data, coords, and time
+    '''
+    time = []
+    coords = []
+    data = []
+    file_prefix = args["file_prefix"]
+    filename_list = args["filename_list"]
+    var_name = args["var"]
+    h5files = get_h5_files(file_prefix, filename_list)
+    volfiles = [h5file['element_data.vol'] for h5file in h5files]
+    # Get a list of times from the first vol file
+    ids_times = [(obs_id, volfiles[0][obs_id].attrs['observation_value'])
+                 for obs_id in volfiles[0].keys()]
+    ids_times.sort(key=lambda pair: pair[1])
+    time_data = {}
+    for id_and_time in ids_times:
+        time_data[id_and_time] = []
+        local_data = []
+        local_coords = []
+        for volfile in volfiles:
+            obs_id_file = volfile[id_and_time[0]]
+            local_data += list(obs_id_file[var_name])
+            local_coords += list(obs_id_file['InertialCoordinates_x'])
+        local_coords_and_data = zip(local_coords, local_data)
+        time_data[id_and_time] += local_coords_and_data
+
+        time_data[id_and_time].sort(key=lambda pair: pair[0])
+        time.append(id_and_time[1])
+        coords.append([pair[0] for pair in time_data[id_and_time]])
+        data.append([pair[1] for pair in time_data[id_and_time]])
```

Comment:
you're appending to the `time_data` dictionary but never use it. I think you can just delete `time_data`. Also there's a lot of list manipulation going on, some of which is unnecessary. I suggest this:

```suggestion
        local_data = np.array(volfiles[0][obs_id][var_name])
        local_coords = np.array(volfiles[0][obs_id]['InertialCoordinates_x'])
        for volfile in volfiles[1:]:
            local_data = np.hstack([local_data, volfile[obs_id][var_name]])
            local_coords = np.hstack(
                [local_coords, volfile[obs_id]['InertialCoordinates_x']])
        ordering = np.argsort(local_coords)
        time.append(local_time)
        coords.append(local_coords[ordering])
        data.append(local_data[ordering])
```